### DISCUSSION TITLE
why dont the doc. mention ,to prefer rust if "formal-verification"(at cost of compile-time) is the goal

### DISCUSSION THREAD

**Discussion by gintominto5329** [carbon-language/carbon-lang#3606] [General]
thanks

**Comment by chandlerc** [comment]
Our docs do suggest to prefer Rust if it works for you in the "[why not rust]" FAQ entry.

[why not rust]: docs/project/faq.md#why-not-rust

If you have a specific suggestion for an improvement to that entries wording, a PR would probably be the most clear way to articulate what could be improved?
